app/ml_models/controllers.py

```diff
@@ -218,9 +218,11 @@ def train_model():
     # plt.show()
     # TODO: Rework this train_model function to focus on training and saving models
     # Fit the model to some data
+    pipeline = Pipeline([('scaler',scaler), ('estimator',estimator8)])
```

Comment:
The StandardScaler scales the columns of an array it's given based on the mean and variance of the column. This has the effect of centering and smoothing out the distribution of values for the column. In our case, we don't use the centering feature, because it would destroy the sparseness of our data. Scaling everything in a column to that column's variance, however, means the values now fit in a narrower range and now are less exaggerated if they were previously close to the extremes.